tests/phpunit/tests/test-static-pages.php

```diff
@@ -86,20 +84,19 @@ public function set_up() {
 	}
 
 	private function init_test( $env = 'frontend' ) {
-		$pll_admin = new PLL_Admin( $this->links_model );
-		$pll_admin->links = new PLL_Admin_Links( $pll_admin );
-		$pll_admin->static_pages = new PLL_Admin_Static_Pages( $pll_admin );
-
 		update_option( 'show_on_front', 'page' );
 		update_option( 'page_on_front', self::$home_fr );
 		update_option( 'page_for_posts', self::$posts_fr );
 
+		self::$model->clean_languages_cache();
+		$pll_admin = new PLL_Admin( $this->links_model );
+		$pll_admin->links = new PLL_Admin_Links( $pll_admin );
+
```

Comment:
We shouldn't need to specifically clean the language cache as [this is done when updating the options above](https://github.com/polylang/polylang/blob/84bcffcab9eb235930510ea1342fd4e9c76028b6/include/static-pages.php#L65-L67). 

Probably you are obliged to that because you instantiate the class after instead of before updating the options.